This fixes "margin colour" (colour used to clear margins - e.g. a half of line
at the bottom of 100x37 console on 800x600 framebuffer).

I don't know what was the intention behind using attr_bgcol_ec() here, but it
caused using of background colour of last erase character to clear margins -
which definitely isn't what we want...
This patch changes margin colour to black (or colour 0 in palette modes).

	-- Jakub Bogusz <qboosh@pld-linux.org>

--- linux-2.6.0-test2/drivers/video/console/fbcon.c.orig	2003-07-14 05:36:32.000000000 +0200
+++ linux-2.6.0-test2/drivers/video/console/fbcon.c	2003-07-31 00:53:26.000000000 +0200
@@ -502,7 +502,7 @@
 	unsigned int bs = info->var.yres - bh;
 	struct fb_fillrect region;
 
-	region.color = attr_bgcol_ec(bgshift, vc);
+	region.color = 0;
 	region.rop = ROP_COPY;
 
 	if (rw && !bottom_only) {
